instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/patches/client.rb

```diff
@@ -10,6 +10,15 @@ module Trilogy
       module Patches
         # Module to prepend to Trilogy for instrumentation
         module Client
+          module StringSetter
+            extend self
+
+            def set(carrier, key, value)
+              encoded = Base64.strict_encode64("\"#{key}\":\"#{value}\"}")
+              carrier.gsub!(/\A/, "/*VT_SPAN_CONTEXT=#{encoded}*/")
```

Comment:
The format should be configurable, this is too specific as downstream sql clients might have different expectations for trace context formatting.

Reply:
👍 I think this needs to be extracted to a Vitess propagator that wraps the Uber propagator.